### ISSUE TITLE
Feature: Wildcard support for Counter instances

### ISSUE THREAD

**Issue by lazi3b0y** [LAB02-Research/HASS.Agent#192]
Please make sure your feature isn't already listed on HASS.Agent's issue tracker:

https://lab02research.youtrack.cloud/

There's a list of open bugs & features on the right. On the left you can see what's being worked on, what's planned and what's ready for the next release.

If you're not sure, don't worry, just go ahead and we'll discuss :)

----

**Is your feature request related to a problem? Please describe.**
Not really, would just save me time and make life easier. :) 
Since I'm sending all this data to my influxdb instance I can then more effectively filter among the sensors and get the data I'd want that way to present it in grafana.

**Describe the solution you'd like**
I'd like to be able to, for example, use a wildcard to select every instance for a Counter when adding performance monitor sensors. This would ideally create one sensor for every instance. Would also be nice if you could use the wildcard to select instances than contains certain string values, i.e. *_3D would let me select every gpu instance that ends with _3D

**Describe alternatives you've considered**
Can't think of any alternatives, besides adding every instance by hand, which would take a very long time, especially when adding things like gpu sensors, since they have very long names with a mix of what looks like memory addresses and categories, i.e. pid_28104_luid_0x00000000_0x0000ECFD_phys_0_eng_2_engtype_VideoDecode

**Additional context**
Not that I can think of. :)

**Comment by LAB02-Admin** [comment]
Good idea, added to the list: [hassagent-154](https://lab02research.youtrack.cloud/issue/hassagent-154)
